demo/video_demo.py

```diff
@@ -19,8 +19,8 @@ def parse_args():
     parser.add_argument('--out', type=str, help='Output video file')
     parser.add_argument('--show', action='store_true', help='Show video')
     parser.add_argument(
-        '--wait-time',
```

Comment:
MMDetection use '-' to separate words in command arguments, like resume-from, wait-time to ease the usages. So the change in line 22 should be omitted.

Reply:
Oh, absolutely, you are right! But, there is a situation that when I see the code "args.resume_from", I don't know what the argument is and then I want to jump to the parser.add_argument(...) for more information about the args.reume_from, so in the vim normal mode, I will key "mm*ggn" to jump to the destination code line. If the using the _ to separate the words, I can easily jump to line of the code parser.add_argument, but I can't do that if using the - to separate words.

Reply:
I see... The original purpose is to ease the typing in the command line, which we think might be more commonly needed. For your case with VIM, you may directly search for 'wait' rather than 'wait_time'?
For now, we also do not wish break the compatibility of arguments actually